README.md

```diff
@@ -32,6 +32,7 @@ For the best experience, you should upgrade your Nexus Repository Manager and Go
 
 | Nexus Repository Manager 3 Version | Google Cloud Storage Blobstore Version |
 | ---------------------------------- |--------------------------------------- |
+| 3.30                               | [Get 0.19.2](https://repo1.maven.org/maven2/org/sonatype/nexus/plugins/nexus-blobstore-google-cloud/0.19.1/nexus-blobstore-google-cloud-0.19.2.kar) |
```

Comment:
I typically don't update this line until after the release is completed (I update this and the Dockerfile after).

Reply:
Ah, just saw this comment. 

FWIW, I've used the `LATEST` in docker stuff to avoid the maintenance of version bumping. e.g. https://github.com/sonatype-nexus-community/nexus-repository-apk/blob/master/Dockerfile#L3